tests/integration/test_api.py

```diff
@@ -381,3 +387,64 @@ def test_annotation_event(client, monkeypatch, httpserver):
             },
         )
     assert result.status_code == 200
+
+
+def test_prediction_collection_no_filter(client):
+    result = client.simulate_get("/api/v1/predictions/")
+    assert result.status_code == 200
+    assert result.json == {"count": 0, "predictions": [], "status": "no_predictions"}
+
+    prediction1 = PredictionFactory(value_tag="en:seeds")
+    result = client.simulate_get("/api/v1/predictions/")
+    assert result.status_code == 200
+    data = result.json
+    assert data["count"] == 1
+    assert data["status"] == "found"
+    prediction_data = data["predictions"]
+    assert prediction_data[0]["id"] == prediction1.id
+    assert prediction_data[0]["type"] == "category"
+    assert prediction_data[0]["value_tag"] == "en:seeds"
+
+    prediction2 = PredictionFactory(
+        value_tag="en:beers", data={"sample": 1}, type="brand"
+    )
+    result = client.simulate_get("/api/v1/predictions/")
+    assert result.status_code == 200
+    data = result.json
+    assert data["count"] == 2
+    assert data["status"] == "found"
+    prediction_data = data["predictions"]
+    assert prediction_data[1]["id"] == prediction2.id
+    assert prediction_data[1]["type"] == "brand"
+    assert prediction_data[1]["value_tag"] == "en:beers"
+
+
+def test_get_predictions():
+    PredictionFactory(barcode="123", keep_types="category", value_tag="en:seeds")
+    PredictionFactory(type="category")
+    PredictionFactory(value_tag="en:seeds")
+    PredictionFactory(type="brand")
```

Comment:
I would like those creations to reflect the data proposal here : https://github.com/openfoodfacts/robotoff/pull/815#discussion_r923060687

More over it's better to keep created predictions in variables (prediction1, prediction2…), so that you can check for ids in you test (first request you assert the fields, but then you just assert ids).